schema/codelists/closed/metric_type.csv

```diff
@@ -0,0 +1,6 @@
+Code,Title,Definition
```

Comment:
```suggestion
Code,Title,Description
```

Reply:
All of the other codelists use Definition not Description

Reply:
Good point :-)